gravitee-apim-console-webui/src/components/gio-ee-unlock-dialog/gio-ee-unlock-dialog.component.html

```diff
@@ -23,13 +23,30 @@
 </div>
 
 <mat-dialog-content class="gio-ee-unlock-dialog__content">
-  <img class="gio-ee-unlock-dialog__content__image" [src]="featureMoreInformation.image" [alt]="'Unlock Gravitee Enterprise'" />
-  <h2 class="gio-ee-unlock-dialog__content__title">Unlock Gravitee Enterprise</h2>
+  <img
+    class="gio-ee-unlock-dialog__content__image"
+    [src]="featureMoreInformation.image"
+    [alt]="featureMoreInformation.title || 'Unlock Gravitee Enterprise'"
+  />
+  <h2
+    class="gio-ee-unlock-dialog__content__title"
+    [innerHTML]="featureMoreInformation.title || 'Unlock Gravitee Enterprise' | safe : 'html'"
+  ></h2>
   <div class="gio-ee-unlock-dialog__content__description" [innerHTML]="featureMoreInformation.description | safe : 'html'"></div>
   <div class="gio-ee-unlock-dialog__content__action">
-    <button class="gio-ee-unlock-dialog__content__action__button" mat-button mat-stroked-button color="accent">
+    <a
+      class="gio-ee-unlock-dialog__content__action__button"
+      mat-button
+      mat-stroked-button
+      color="accent"
+      target="_blank"
+      [href]="
+        'https://www.gravitee.io/contact-us-alerts?utm_source=oss_apim&utm_campaign=oss_apim_to_ee_apim&utm_medium=' +
```

Comment:
Shouldn't you use the link from GioLicenseService > ctaLinkPrefixa?

Reply:
There's another PR coming to handle that here https://github.com/gravitee-io/gravitee-api-management/pull/4552